README.md

```diff
@@ -13,8 +13,11 @@ SQL recipes for Data Export analysis are in the [Data Export Cookbook](https://g
 ## Quick Start
 * Make sure you have [installed](https://golang.org/doc/install) Go 1.9 or higher.
 * Build it (for EC2, for example): ``GOOS=linux GOARCH=amd64 go get github.com/fullstorydev/hauser``
+    - Type `go version` to find your `GOOS` and `GOARCH` values.
+    - Example (macOS): `go1.11.5 darwin/amd64` is `GOOS=darwin GOARCH=amd64`
 * Copy the included `example-config.toml` file and customize it for your environment, including your FullStory API key, warehouse host, and credentials. AWS credentials (for S3) come from your local environment.
-* Run it: `./hauser -c <your updated config file>`
+* Run it: `$GOPATH/bin/hauser -c <your updated config file>`
```

Comment:
We may need to go deeper down the rabbit hole of people not knowing Go here; this change assumes that your GOPATH is set. If it's not set it has a [default value](https://golang.org/doc/code.html#GOPATH), so the `go get` command will succeed, but we'll still fail to run here. (as you can see I found all the official Go docs on this subject since we last talked!)